### DISCUSSION TITLE
Cannot find module 'regexp-to-ast'

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-langium/langium#1718] [Q&A] [answered]
environment: "langium": "^1.1.0"
i try to package my project as a  vscode plugin,
i encapsulate CLI command as a vscode command like below,
![image](https://github.com/user-attachments/assets/0cb84196-1fa3-4de8-b9aa-c640f242014f)

when i call this command from other project ,
![image](https://github.com/user-attachments/assets/0025f6a5-d62c-49fb-b1d4-adb10c75df5e)
an error occured
![image](https://github.com/user-attachments/assets/3abaedca-aecc-437d-9886-af371ce1e892)

**Comment by msujew** [comment, marked as answer]
Hey @zrwsmd,

you should bundle everything in your vscode extension. That way you don't need to rely on some dependencies included in the `node_modules` of your vscode extension.

See also our [bundling guide](https://langium.org/docs/recipes/code-bundling/).
